docs/guides/building/cross-platform.md

```diff
@@ -251,6 +251,10 @@ Keep in mind that this file is specific to your machine so we don't recommend ch
 
 #### Building the App
 
+If your application has dependencies has bindings to C libraries such as `ring` or `libsqlite3-sys`, building your application cross-platform gets a little trickier, as you will also need to set up the 
```

Comment:
If we don't have a specific thing we're telling the reader to do (such as set this env var value to "xyz") then I like to provide a link to a reference the reader could find that information at (or at least a place to learn more information). Maybe we could simply say "please consult the documentation for those libraries to learn more about any additional configuration required"

Reply:
So, I wanted to give links to specific GitHub issues that I found while searching for it. The problem is that this kind of thing is truly highly variable on the crates that a user is compiling, and there isn't a good single reference for cross-compiling on Rust.

I added your suggestion regarding the last comment